client/src/components/SlideingNav/SlidingNav.js

```diff
@@ -7,72 +7,108 @@ import PostMessageModal from "../Modals/PostMessageModal";
 import PickFormModal from "../Modals/PickFormModal";
 
 const hideStyle = {
-  display:"none"
-}
+  display: "none",
+};
 
-class SlidingNav extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      showModal:""
-    };
-  }
+const SlidingNav = ({ toggleDisplay, toggleSlidingDiv, userObj, logOut }) => {
+  const initialState = {
+    showModal: "",
+  };
```

Comment:
i think this can be removed. i see its passed into the useState below. there you could just initialize it with an empty string